### DISCUSSION TITLE
Are BaseSystem.dmg macOS images supported?

### DISCUSSION THREAD

**Discussion by yukiterunamaana** [quickemu-project/quickemu#931] [Quickemu] [answered]
I don't have a recovery image on my hands ATM, please tell me if that will suffice

**Comment by lj3954** [comment, marked as answer]
I don't understand what you're trying to say. Quickget automatically downloads the macOS Recovery Image from Apple's servers. There is no need for you to provide your own Recovery Image. If quickget is unable to download the recovery image, please file an issue instead.
